db_checker.py

```diff
@@ -92,6 +95,23 @@ def test_interest_units(self):
                         print "ERROR: Missing units on " + err_src_fmt(db, rec)
 
         self.assertEqual(err, 0, msg="Missing units on interesting PVs in project")
+		
+	def test_interest_calc_readonly(self):
+		"""
+		This method checks that interesting PVs have units
+		"""
+        err = 0
+
+        for db in dbs:
+            for rec in db.records:
+                if rec.is_interest() and not rec.is_disable() and (rec.get_type() in ASG_list):
```

Comment:
This was copied from the unit check record above. Does it also need to be removed from `test_interest_units`